### PR TITLE
PR #76 and added background permission

### PR DESCRIPTION
**Pull request by jijojosephk**

### CODE REVIEW

**Comment by jijojosephk** [comment]
1. @bbhtt, I kept the `pipewire` and rest of `dconf` removed as you mentioned in PR #76. Thanks a lot for this.
2. @andreaippo, Added background permission.

Can you both kindly test this?
`flatpak install --user https://dl.flathub.org/build-repo/73491/com.github.IsmaelMartinez.teams_for_linux.flatpakref`

File: com.github.IsmaelMartinez.teams_for_linux.yml [hunk 1]
```diff
@@ -14,17 +14,14 @@ finish-args:
...
   - --filesystem=xdg-run/pipewire-0
-  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
-  - --talk-name=ca.desrt.dconf
   - --talk-name=org.kde.StatusNotifierWatcher
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.freedesktop.PowerManagement
   - --talk-name=org.freedesktop.ScreenSaver
   - --talk-name=org.gtk.Notifications
   - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.portal.Background
```
Comment (bbhtt) [inline com.github.IsmaelMartinez.teams_for_linux.yml:24]: This is not needed, flatpak grants wildcard access to `org.freedesktop.portal.*` just run it with --log-session-bus and you'll see in the top

**Comment by bbhtt** [comment]
Otherwise is fine, thanks.

File: com.github.IsmaelMartinez.teams_for_linux.yml (hunk 1 above)
Comment (jijojosephk) [reply com.github.IsmaelMartinez.teams_for_linux.yml:24]: I see, but there was a request to do so here: https://github.com/IsmaelMartinez/teams-for-linux/issues/1032#issuecomment-1871962083

Comment (bbhtt) [reply com.github.IsmaelMartinez.teams_for_linux.yml:24]: they are wrong unfortunately. Flatseal does not use this permission and the background toggle in Flatseal is not for this.

The background toggle updates the permission store and sets background to yes. That means you allowed it to run in background.

**Pull request merged**